standards/WIT-SVID.md

```diff
@@ -0,0 +1,127 @@
+
+TODO: Should we summarize `alg`/`typ` even if these will not differ from the WIMSE WIT?
+
+### 2.1. Key ID - `kid`
+
+The `kid` header is defined by the [JSON Web Signature (JWS)][2] document.
+
+For a WIT-SVID, this header is mandatory. This differs from the upstream WIT itself where this header is optional.
+
+The precise structure of this header is unspecified, and, it MUST be treated by verifiers as a case-sensitive string.
+
+The issuer MUST ensure that the value set within the `kid` header is unique to each issuing key-pair.
+
+### 2.2. Additional Headers
+
+It is permitted for an implementation to include additional headers not specified in this document or the upstream document.
+
+### 3. Claims
+
+This section describes the claims that may be included in the WIT-SVID.
+
+The WIT-SVID specification does not introduce any claims beyond those defined by the upstream document for the WIT. However, it does set additional restrictions and provide SPIFFE-specific guidance on some claims.
+
+TODO:
+
+- `iss` claim. This is RECOMMENDED in the WIT specification. Do we wish to mirror this, or, make it mandatory?
+
+### 3.1. Subject - `sub`
+
+The `sub` claim MUST be present and set to the SPIFFE ID of the workload to which it is issued. This is the primary claim against which workload identity is asserted.
+
+For example: `spiffe://example.org/service`.
+
+### 3.2. JWT ID - `jti`
+
+The `jti` claim provides a unique identifier for the WIT-SVID. It is defined by [RFC 7519][6].
+
+Primarily, this claim enables distinguishing one or more WIT-SVIDs that contain the same SPIFFE ID for the purposes of auditing. For example, if a validator of a WIT-SVID records the JTI within an audit log event, this audit log event can be correlated with the one emitted by the issuer, allowing the lineage of the credential to be ascertained.
+
+TODO: Do we repeat from RFC 7519 that the issuer MUST ensure that this field is unique?
+TODO: In WIT-SVID, is this a MAY/SHOULD/MUST?
+TODO: Do we make any comment here on revocation?
+
+### 3.2. Additional Claims
+
+It is permitted for an implementation to include additional claims not specified in this document or the upstream document.
+
+## 4. Token Signing and Validation
+
+## 5. Token Presentation
+
+This section describes the manner in which a WIT-SVID may be presented from one workload to another for the purposes of authentication.
+
+The WIT-SVID MUST always be presented by the workload with proof of possession of the key-pair contained within the `cnf`. In other words, the WIT-SVID MUST NOT be presented as a bearer token.
+
+WIMSE defines protocols for presentation of the WIT. It is recommended that that an implementor use one of these defined protocols, however, it is not required. At the time of writing, there are two protocols:
```

Comment:
But it must be a reliable and compatible PoP mechanism. I don't think we can easily say that "it is not required". Perhaps a different wording could be used. Not sure for now.